pkg/scheduler/plugins/conformance/conformance_test.go

```diff
@@ -0,0 +1,63 @@
+package conformance
+
+import (
+	"reflect"
+	"testing"
+
+	v1 "k8s.io/api/core/v1"
+
+	"volcano.sh/volcano/pkg/scheduler/api"
+	"volcano.sh/volcano/pkg/scheduler/conf"
+	"volcano.sh/volcano/pkg/scheduler/framework"
+	"volcano.sh/volcano/pkg/scheduler/uthelper"
+	"volcano.sh/volcano/pkg/scheduler/util"
+)
+
+func TestConformancePlugin(t *testing.T) {
+	tests := []struct {
+		name string
+		uthelper.TestCommonStruct
+		arguments  framework.Arguments
+		preemptees []*api.TaskInfo
+	}{
+		{
+			name: "conformance plugin",
+			TestCommonStruct: uthelper.TestCommonStruct{
+				Plugins: map[string]framework.PluginBuilder{PluginName: New},
+			},
+		},
+	}
+
+	trueValue := true
+	for _, test := range tests {
+		tiers := []conf.Tier{
+			{
+				Plugins: []conf.PluginOption{
+					{
+						Name:               PluginName,
+						EnabledPreemptable: &trueValue,
+						Arguments:          test.arguments,
+					},
+				},
+			},
+		}
+		// prepare preemptor and preemptees
+		task1 := api.NewTaskInfo(
```

Comment:
we can put common code outside the for loop.

Reply:
make sense! done